packages/@react-spectrum/breadcrumbs/stories/Breadcrumbs.stories.tsx

```diff
@@ -18,7 +18,8 @@ import React from 'react';
 import {storiesOf} from '@storybook/react';
 
 let styles = {
-  width: '100vw'
+  width: '100vw',
+  'padding-inline-start': '30px'
```

Comment:
Not supported in edge.

```suggestion
  'padding': '0 30px'
```